packages/mos-gateway/src/mosHandler.ts

```diff
@@ -420,6 +420,7 @@ export class MosHandler {
 			for (const [deviceId, device] of Object.entries<{ options: MosDeviceConfig }>(devices)) {
 				if (device) {
 					if (device.options.secondary) {
+						this._hotStandby = device.options.secondary?.hotStandby || false
```

Comment:
This isn't safe.
If there are multi sub-devices on this gateway, they will be fighting over `this._hotStandby`, with the value for all of them being taken from the 'last' sub-device

Reply:
I've changed it from being a boolean for all subdevices to be a record pr. subdevice in mosHandler. 
And the naming has been changed from hotStandby to openMediahotStandby